### ISSUE TITLE
[SUGGESTION] Z-Moves Implements? + Mega, and Gigantamax

### ISSUE THREAD

**Issue by AnnoyingPriannacompy** [pokeclicker/pokeclicker#1635]
**Current Behavior**

Nothing, Doesn't Exist Yet But they exist in the main games and Series.

**Improved Behavior**

Well, Gigantamax and Mega-evolutions would be an obvious way to increase the Pokedex. But Z-moves do not give new forms, so I'd like to see the gems or Shards of Z-moves included, to Create Crits or Random Crits, this way some players could enjoy Randomly Crit hitting pokemon, for some more dynamic Actions.

**Reason for change**

Most Clicker Games give Critical hits, of Some sort especially with such a large roster, and To make sure every group of the Franchise can be happy with their versions of pokemon, Z-moves, Gigatamax, And Mega-evolution could be a Gateway to Uniting Our Community. But Most of All, Z-moves are like Ultimate moves, that Only one Type Benefits from it. or very specific Pokemon.

**Detailed Description**

Allow the player to get Z-moves in Alola, either in shops, gyms, Battle prizes, or However you'd get them. They would collected each only once, and it would make it interesting as you already have Shards which improve attacks, Adding Z-move Gems, would allow even more powerful moves.
Gigantamax and Mega-evolutions would just be a Power upgrade. Z-moves could be game changing.

**Screenshots:**

**Comment by anapple96** [comment]
Duplicate of #845 #814 #1039

**Comment by AnnoyingPriannacompy** [comment]
> Duplicate of #845 #814 #1039

I can See the First one, but the other two I do not, I was Just Making a Pass Reference to Mega-evolution, and Dynamax, I genuinely think Z-moves could implemented like I suggested.

**Comment by GarrikFel** [comment]
This might make sense with the new party system, but as it is right now (where every pokemon participates in every fight), it seems like a fair amount of work for basically no payoff.  Now, if you had to limit your team to just 6 pokemon, then z-moves, mega evolutions, and lazy mega evolutions (read: Gigantamax) would have a lot more utility.  Maybe in the future...

**Comment by AnnoyingPriannacompy** [comment]
> This might make sense with the new party system, but as it is right now (where every pokemon participates in every fight), it seems like a fair amount of work for basically no payoff. Now, if you had to limit your team to just 6 pokemon, then z-moves, mega evolutions, and lazy mega evolutions (read: Gigantamax) would have a lot more utility. Maybe in the future...

Ah I see this makes more sense. Thank you.

**Comment by Ultima1990** [comment]
Party is dead.

**Comment by AnnoyingPriannacompy** [comment]
YAY!

**Comment by Aegyo** [comment]
Z Moves was approved in #3556
Megas are being added now
Gigantamax approved in #3796

Closing this issue as duplicate
